moped-database/migrations/1696268106764_alter_table_public_moped_phases_add_column_phase_name_simple/up.sql

```diff
@@ -0,0 +1,13 @@
+alter table moped_phases add column "phase_name_simple" text null;
+
+UPDATE moped_phases SET phase_name_simple = 'Complete' WHERE phase_id in (1, 2);
```

Comment:
Took a quick peek, and it looks like these two should have different simple names. I was checking my local DB against the table in the issue.

Reply:
yup! I should have stuck a WIP on this after the table in the issue changed, but i was sucked into the mapbox draw controls and couldnt escape. its updated now.